tests/SampleApp/SampleApp/CMakeLists.txt

```diff
@@ -1,6 +1,5 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
-include(add_enclave_executable)
 add_executable(SampleApp CallWrappers.cpp SampleApp.cpp)
```

Comment:
Wait... shouldn't this now (and the rest below) be using `add_enclave` but without signing? (Which above should handle linking to `oeenclave` if needed). That is, I think we should be undoing the move to `add_executable` that @anakrish was previously forced to make.